back/boxtribute_server/mobile_distribution/crud.py

```diff
@@ -92,6 +95,88 @@ def change_distribution_event_state(distribution_event_id, distribution_event_st
     return distribution_event
 
 
+def set_products_for_packing_list(
+    user_id, distribution_event_id, product_ids_to_add, product_ids_to_remove
+):
+    """
+    Set the products for a packing list.
```

Comment:
The docstring is identical to the function name. I suggest we omit it, or explain the implementation below in a bit more detail (moving comments like `remove/add new products` into the docstring)

Reply:
Good point. I would even think, once we do a proper code cleanup of the new features, to make functions much shorter and extract things like "Remove products" and "Add products" into seperate functions with proper names (this would also remove the need for more descriptive comments).